src/store/storage/vscode-storage.ts

```diff
@@ -4,7 +4,7 @@ export function createVscodeStorage(memento: Memento) {
     return {
         getItem: (key: string): Promise<string> =>
             new Promise((resolve, _reject) => {
-                resolve(memento.get(key));
+                resolve(memento.get(key) as any);
```

Comment:
This should probably just specify the same type as it's method like so:

```suggestion
                resolve(memento.get<string>(key) ?? "");
```

Since memento.get can return undefined for non-existent entries we need to provide a default.